Controller/RegistrationController.php

```diff
@@ -74,6 +75,54 @@ public function registerAction(Request $request)
     }
 
     /**
+     * Resend the confirmation email to the User
+     */
+    public function resendConfirmAction(Request $request)
+    {
+        /** @var $formFactory \FOS\UserBundle\Form\Factory\FactoryInterface */
+        $formFactory = $this->container->get('fos_user.resend_confirm.form.factory');
+        /** @var $userManager \FOS\UserBundle\Model\UserManagerInterface */
+        $userManager = $this->container->get('fos_user.user_manager');
+        /** @var $dispatcher \Symfony\Component\EventDispatcher\EventDispatcherInterface */
+        $dispatcher = $this->container->get('event_dispatcher');
+
+        $user = $userManager->createUser();
+
+        $form = $formFactory->createForm();
+        $form->setData($user);
+
+        if ('POST' === $request->getMethod()) {
+            $form->bind($request);
+
+            if ($form->isValid()) {
+                $email = $user->getEmail();
+                $user = $userManager->findUserByEmail($email);
+
+                if (null === $user) {
+                    throw new NotFoundHttpException(sprintf('The user with email "%s" does not exist', $email));
```

Comment:
@merk It can't be a security hole to show whether an email address is attached to an existing account, as it's not a secret: anyone can try to create an account using any email address, and it will fail if there is already an account with that email address.